addons/im_livechat/models/chatbot_script_step.py

```diff
@@ -346,7 +346,8 @@ def _process_step_forward_operator(self, discuss_channel):
             # then post a small custom 'Operator has joined' notification
             discuss_channel._chatbot_post_message(
                 self.chatbot_script_id,
-                Markup('<div class="o_mail_notification">%s</div>') % _('%s has joined', human_operator.partner_id.name))
+                Markup('<div class="o_mail_notification">%s</div>')
+                % _('%s has joined', human_operator.livechat_username or human_operator.partner_id.name))
```

Comment:
Didn't we have another PR elsewhere that change "has joined" to "joined" and similar sutff?

Reply:
It did not involved this message but I guess we could also adapt this one :man_shrugging: